docs/reference/search/profile.asciidoc

```diff
@@ -276,7 +282,7 @@ the equivalent name in Elasticsearch.  The `"description"` field displays the Lu
 is made available to help differentiating between parts of your query (e.g. both `"message:search"` and `"message:test"`
 are TermQuery's and would appear identical otherwise.
 
-The `"time"` field shows that this query took ~15ms for the entire BooleanQuery to execute.  The recorded time is inclusive
+The `"time_in_nanos"` field shows that this query took ~1.8ms for the entire BooleanQuery to execute.  The recorded time is inclusive
```

Comment:
In all likelihood just a typo :)  I generated those by running the query and c/p the results.  But it changed a few times, so I probably just missed it during one of the changes